pkg/ui/src/InventoryTable/InventoryTableRow.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+	import type { TableData } from './types';
+	export let isbn;
+	export let title;
+	export let author;
+	export let editor;
+	export let quantity;
+	export let price;
+	export let year;
+	export let outOfPrint;
+	export let editedBy;
+	export let classification;
+
+	const cols: TableData = {
+		isbn,
+		title,
+		author,
+		editor,
+		quantity,
+		price,
+		year,
+		outOfPrint,
+		editedBy,
+		classification
+	};
+</script>
+
+<tr class="even:bg-gray-50 whitespace-nowrap text-sm text-gray-500 ">
+	{#each Object.values(cols) as col}
+		<td class="first:text-gray-900 first:font-medium py-2 pl-4">
+			{col || 'N/A'}
```

Comment:
Let's check if it's `undefined` and then use "N/A" as out of print = false will show "N/A" for instance

Reply:
You can use `col ?? 'N/A'`